wies/core/views.py

```diff
@@ -290,12 +401,38 @@ def get_context_data(self, **kwargs):
         else:
             context['next_page_url'] = None
 
+        # Panel state is now handled by HTMX in get() method
```

Comment:
this comment does not make sense. i believe its a remnant of previous code changes that are no longer here